src/main.js

```diff
@@ -32,6 +32,8 @@ import Nextcloud from './mixins/Nextcloud'
 import router from './router'
 import store from './store'
 import { fixAccountId } from './service/AccountService'
+// Styles
+import '@nextcloud/dialogs/styles/toast.scss'
```

Comment:
oh, this is about toasts, no? There is no relation to the popover from `@nextcloud/vue`